src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Page.al

```diff
@@ -0,0 +1,319 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+using System.Environment;
+using System.TestTools.TestRunner;
+
+page 149031 "AIT Test Suite"
+{
+    Caption = 'AI Test Suite';
+    PageType = Document;
+    SourceTable = "AIT Test Suite";
+    Extensible = false;
+    DataCaptionExpression = this.PageCaptionLbl + ' - ' + Rec."Code";
+
+    layout
+    {
+        area(content)
+        {
+            group(General)
+            {
+                Caption = 'AI Test Suite';
+                Enabled = Rec.Status <> Rec.Status::Running;
+
+                field("Code"; Rec."Code")
+                {
+                    ToolTip = 'Specifies the ID of the test suite.';
+                    ApplicationArea = All;
```

Comment:
ApplicationArea can be moved to page level

Tooltips can be moved to record level